### DISCUSSION TITLE
v0.13.0

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/steamguard-cli#373] [General]
## What's Changed

* rename `trade` subcommand to `confirm` and deprecate `trade` subcommand by @dyc3 in https://github.com/dyc3/steamguard-cli/pull/368
    * `trade` will still work, but may be removed in a future version
* Bump mio from 0.8.8 to 0.8.11 by @dependabot in https://github.com/dyc3/steamguard-cli/pull/372

**Full Changelog**: https://github.com/dyc3/steamguard-cli/compare/v0.12.6...v0.13.0

<hr /><em>This discussion was created from the release <a href='https://github.com/dyc3/steamguard-cli/releases/tag/v0.13.0'>v0.13.0</a>.</em>

**Comment by xDroflLOL** [comment]
this will definitely not break muscle memory :+1: (i am the outlier)
